### ISSUE TITLE
1

### ISSUE THREAD

**Issue by gooobot** [JiauZhang/tracking-paper#4]
DeltaEdit: Exploring Text-free Training for Text-Driven Image Manipulation
FaceChat: An Emotion-Aware Face-to-face Dialogue Framework
DS-Fusion: Artistic Typography via Discriminated and Stylized Diffusion
GlueGen: Plug and Play Multi-modal Encoders for X-to-image Generation
DialogPaint: A Dialog-based Image Editing Model
https://github.com/pytorch/pytorch/blob/master/torch/utils/tensorboard/_pytorch_graph.py
NeRF-LOAM: Neural Implicit Representation for Large-Scale Incremental LiDAR Odometry and Mapping
ChatGPT for Programming Numerical Methods
Is ChatGPT A Good Keyphrase Generator? A Preliminary Study
Zero-guidance Segmentation Using Zero Segment Labels
Text2Video-Zero: Text-to-Image Diffusion Models are Zero-Shot Video Generators
CoBIT: A Contrastive Bi-directional Image-Text Generation Model
ReVersion: Diffusion-Based Relation Inversion from Images
Make-It-3D: High-Fidelity 3D Creation from A Single Image with Diffusion Prior
MindDiffuser: Controlled Image Reconstruction from Human Brain Activity with Semantic and Structural Diffusion
ChatDoctor: A Medical Chat Model Fine-tuned on LLaMA Model using Medical Domain Knowledge
Conditional Image-to-Video Generation with Latent Flow Diffusion Models
High Fidelity Image Synthesis With Deep VAEs In Latent Space
AgileGAN3D: Few-Shot 3D Portrait Stylization by Augmented Transfer Learning
Masked Diffusion Transformer is a Strong Image Synthesizer
Freestyle Layout-to-Image Synthesis
Better Aligning Text-to-Image Models with Human Preference
Training-free Style Transfer Emerges from h-space in Diffusion models
OmniAvatar: Geometry-Guided Controllable 3D Head Synthesis
StyleDiffusion: Prompt-Embedding Inversion for Text-Based Editing
Prompt-to-Prompt Image Editing with Cross Attention Control
Null-text Inversion for Editing Real Images using Guided Diffusion Models
Robust Dancer: Long-term 3D Dance Synthesis Using Unpaired Data
MDP: A Generalized Framework for Text-Guided Image Editing by Manipulating the Diffusion Path
Masked and Adaptive Transformer for Exemplar Based Image Translation
PAIR-Diffusion: Object-Level Image Editing with Structure-and-Appearance Paired Diffusion Models
